quantum/rgb_matrix.c

```diff
@@ -250,12 +250,12 @@ static void rgb_task_timers(void) {
     }
     last_hit_buffer.tick[i] += deltaTime;
   }
-#endif
+#endif // defined(RGB_MATRIX_KEYPRESSES) || defined(RGB_MATRIX_KEYRELEASES)
 }
 
 static void rgb_task_sync(void) {
   // next task
-  if (timer_elapsed32(g_rgb_counters.tick) > 16)
+  if (timer_elapsed32(g_rgb_counters.tick) > RGB_MATRIX_LED_FLUSH_LIMIT)
```

Comment:
Should be >= RGB_MATRIX_LED_FLUSH_LIMIT. Verified using scope.

Reply:
good point

Reply:
fixed